reana-ui/src/pages/profile/components/Quota.js

```diff
@@ -55,9 +57,86 @@ export default function Quota() {
     );
   }
   return (
-    <Grid columns={2}>
-      {renderPieChart("CPU", quota.cpu)}
-      {renderPieChart("HDD", quota.disk)}
-    </Grid>
+    <>
+      <Grid columns={2}>
+        {renderPieChart("CPU", quota.cpu)}
+        {renderPieChart("Disk", quota.disk)}
+      </Grid>
+      <div className={styles["quota-info"]}>
+        {getQuotaNotifications(quota, config.adminEmail).map(
+          (notification, index) => {
+            const message = (
+              <span>
+                {notification.body} Please see:{" "}
+                <a
+                  href={REANA_QUOTAS_DOCS_URL}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {REANA_QUOTAS_DOCS_URL}
+                </a>
```

Comment:
(minor) What do you think about "_For more details, please check out the related [documentation page](https://docs.reana.io/advanced-usage/user-quotas)._" ?